test/no-inheritance.js

```diff
@@ -13,28 +13,71 @@ var userConfig = {
     }
 };
 
+var userConfigNoInterface = {
```

Comment:
please add 1 special test section and pass invalid options to the rule. This ensures that the schema defined for options is strictly followed and nothing unexpected cripples in.
add below tests
`"security/no-inheritance": ["error", { "no-interffff": true }]`
`"security/no-inheritance": ["error", {}]`
`"security/no-inheritance": ["error", { 1: 2 }]`
`"security/no-inheritance": ["error", { "no-interface": null }]`
`"security/no-inheritance": ["error", [true]]`
`"security/no-inheritance": ["error", { "no-interface": [true] }]`
`"security/no-inheritance": ["error", { "": false }]`
`"security/no-inheritance": ["error", { "no-interface": true, extraRandomProperty: "blah" }]`

The only configurations that should pass are
`"security/no-inheritance": ["error", { "no-interface": true }]`
`"security/no-inheritance": ["error", { "no-interface": false }]`
`"security/no-inheritance": ["error"]`

(you can add any other tests that you feel might be relevant)

Reply:
> "security/no-inheritance": ["error", {}]

is considered valid. I did not add it to the tests.